<div class="bannercontainer">
  <img src="http://2018.igem.org/wiki/images/1/1b/T--GO_Paris-Saclay--tableau_banner.png"
       class="bannerimg superposedbanner1">
  <img src="http://2018.igem.org/wiki/images/9/90/T--GO_Paris-Saclay--empty-banner.png"
       class="bannerimg superposedbanner2">
  <div class="headlinetext">HeteroGenious</div>
</div>

<mat-card class="dashboard-card largemargin">
<p class="spacemaker"></p>
  <div class="titrebloc">     <img class="carretitrepincipal" src="http://2018.igem.org/wiki/images/f/fd/T--GO_Paris-Saclay--carretitreprinci.png">     <h1>Context</h1>   </div>

  <p>Non-genetic phenotypic heterogeneity is present in large-scope in bacterial population. In
  enteropathogenic Escherichia coli (EPEC), the expression of the locus of enterocyte effacement (LEE) pathogenicity
  island is controlled by the interplay of Ler (LEE encoded regulator) and H-NS (naturally expressed by all E. coli
  strains). This interplay results in the coexistence of nonvirulent and hypervirulent subpopulations<reference-box shorthand="Leh2017"></reference-box>.</p>
  <mat-card class="normalmargin">
    <svg class="lee5explainer" viewBox="0 0 250 100" width="800px">
      <g
        id="layer1-7"
        inkscape:label="Layer 1"
        transform="translate(0.21045238,11.007044)">
        <path
          inkscape:connector-curvature="0"
          id="DNA_tractline"
          d="M 24.273365,68.217281 H 199.06763"
          style="fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:1.70000005;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
        />
        <rect
          y="59.248367"
          x="82.020828"
          height="15.501633"
          width="105.83334"
          id="GFP_gene"
          style="opacity:1;fill:#50b47b;fill-opacity:1;stroke:none;stroke-width:2.04960823;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1"
        />
        <path
          inkscape:connector-curvature="0"
          id="LEE5_Prom_Arrow"
          d="M 56.126599,68.217279 V 54.051998 h 9.621705"
          style="fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:1.70000005;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1;marker-end:url(#Arrow1Send)"
        />
        <text
         xml:space="preserve"
         style="font-style:italic;font-variant:normal;font-weight:300;font-stretch:normal;font-size:10.58333302px;line-height:125%;font-family:'Source Sans Pro';-inkscape-font-specification:'Source Sans Pro, Light';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;letter-spacing:0px;word-spacing:0px;writing-mode:lr-tb;text-anchor:start;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
         x="126.95313"
         y="70.528824"
         id="GFP_Label"
         ><tspan
           x="126.95313"
           y="70.528824"
           id="tspan2612"
           style="stroke-width:0.26458332px"><tspan
             x="126.95313"
             y="70.528824"
             id="tspan2610"
             style="stroke-width:0.26458332px">GFP</tspan></tspan></text>
      <text
         xml:space="preserve"
         style="font-style:normal;font-weight:normal;font-size:10.58333302px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
         x="46.903778"
         y="77.547478"
         id="LEE5_Label"><tspan
           x="46.903778"
           y="77.547478"
           id="tspan2618"
           style="stroke-width:0.26458332px"><tspan
             x="46.903778"
             y="77.547478"
             style="font-style:italic;font-variant:normal;font-weight:300;font-stretch:normal;font-size:10.58333302px;font-family:'Source Sans Pro';-inkscape-font-specification:'Source Sans Pro, Light Italic';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;writing-mode:lr-tb;text-anchor:start;stroke-width:0.26458332px"
             id="tspan2616">LEE5</tspan></tspan></text>
        <path
          style="fill:#ff3800;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
          d="M 51.645202,28.269915 H 40.589667 l -5.918626,-5.459935 5.806954,-5.047855 h 11.055534 l -5.471928,5.047855 5.471928,5.047857 z"
          id="HNS"/>
        <path
          style="fill:#ff3800;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
          d="M 51.645202,28.269915 H 40.589667 l -5.918626,-5.459935 5.806954,-5.047855 h 11.055534 l -5.471928,5.047855 5.471928,5.047857 z"
          id="HNS2"/>
        <path
          style="fill:#ff3800;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
          d="M 51.645202,28.269915 H 40.589667 l -5.918626,-5.459935 5.806954,-5.047855 h 11.055534 l -5.471928,5.047855 5.471928,5.047857 z"
          id="HNS3"/>
        <path
          style="fill:#ff3800;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
          d="M 51.645202,28.269915 H 40.589667 l -5.918626,-5.459935 5.806954,-5.047855 h 11.055534 l -5.471928,5.047855 5.471928,5.047857 z"
          id="HNS4"/>
        <path
          style="fill:#ee9942;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
          d="m 85.950738,27.608459 h -11.05554 l -5.918624,-5.459936 5.806954,-5.047855 h 11.05553 c 0.08057,3.365533 0.03878,6.730179 0,10.095713 z"
          id="Ler"/>


        <path
          style="fill:#216398;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
          d="m 159.90191,38.635006 h -17.37252 c -8.7785,-1.214539 -7.30392,-5.148532 -9.35443,-8.285353 -8.62849,2.681416 -7.43344,0.162134 -10.69078,0 -5.48854,-9.664339 -5.93783,-14.587134 0.66817,-10.022609 14.65528,0 23.60451,-8.559072 43.96584,0 0.38289,7.845351 -1.89798,14.025855 -7.21628,18.307962 z"
          id="RNA_Pol"/>
        <path
          style="fill:none;fill-opacity:1;fill-rule:evenodd;stroke:#000000;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:12.00000033, 2.00000006;stroke-dashoffset:0;stroke-opacity:1"
          d="M 133.69158,10.972607 120.86266,2.9545174 h -16.43709 v -4.9444776 h 16.43709 l 12.82892,7.4067856 z"
          id="RNA"/>

        <text
          x="32.673473"
          y="14.795038"
          id="HNSLabel"
        >H-NS
        </text>

        <text
          x="71.794014"
          y="13.472123"
          id="LerLabel">Ler
        </text>

        <text
          x="71.794014"
          y="13.472123"
          id="RNAPolLabel">RNA Polymerase
        </text>


      </g>
    </svg>
    <div class="mat-caption captionafterapicture">Figure 1 : Interplay between Ler and H-NS for the modulation of <i>LEE5</i>
      promoter activity
      <br>
      <div class="captionmaterialmethod">
        Ler is a nucleoid associated protein encoded by LEE (locus of enterocyte effacement)
        genomic island encoding a secretion system in enteropathogenic <i>E. coli</i>. For the project, we used
        <i>ler</i> and its target DNA,
        a small part of LEE island containing the promoter region of the 5th operon, named <i>LEE5</i>. This region has
        been shown to
        be recognized by two nucleoid-associated proteins, H-NS and Ler.  H-NS and Ler belong to the same family of
        protein of
        proteins qualified as ‘silencers’ of foreign DNA expression, the H-NS family. However, it has been reported that
        Ler can
        counteract the H-NS silencing on the LEE island.
      </div>
    </div>
  </mat-card>

  <p>We get inspiration from this natural system for creating our HeteroGenious device for the methotrexate (MTX) project.
  The idea was to propose a new genetic system, very simple (only two parts in E. coli) that can easily generate
  phenotypic heterogeneity within a clonal population with some bacteria that will significantly express the
    biotranformation pathway and other that will not and will ensure bacterial renewal within a bioreactor.</p>


  <div class="titrebloc">     <img class="carretitrepincipal" src="http://2018.igem.org/wiki/images/f/fd/T--GO_Paris-Saclay--carretitreprinci.png">     <h1>Toward a ‘HeteroGenious’ expression of BioBricks</h1>   </div>

  <p>
    While analysing the growth of our chassis strain (WT and &Delta;<i>tolC</i> strain) in absence or presence of MTX,
    we noticed that the strains harboring pSB1C3-<i>folC</i>-<i>cpg2</i> present a growth delay/defect.
    More generally, BioBrick expression and/or drug degradation may be toxic for the chassis strains, and can be
    detrimental
    for the long term maintenance of a cleaning bioreactor ;
    anti-cancer drugs degradation pathways may be especially affected, given these drugs target very basic mecanism of
    life,
    likely to be conserved from bacteria to man.
  </p>
  <p>
    Therefore we wanted our bioreactor to harbor a heterogeneous bacterial population so that all of them will not
    express
    the degradation pathway at the same time. This system is so called ‘HeteroGenious’ since it generates ‘ingenious’
    heterogeneity within the ‘cleaning factory’ population.
  </p>

  <p>
    We took advantage of a recently described regulation system that can generate bimodal
    expression patterns in <i>E. coli</i>
    <reference-box shorthand="Leh2017"></reference-box>
    . Only two components must be provided to
    implement this ‘heterogenious’ system in <i>E. coli</i>: <i>LEE5</i> promoter region and the gene encoding
    Ler (‘LEE encoded regulator’). The <i>E. coli</i> endogeneous H-NS silencer also plays a part in the system : the
    competition between H-NS and Ler for <i>LEE5</i> binding can
    generate some heterogeneity of transgene expression within <i>E. coli</i> populations
    <reference-box shorthand="Leh2017"></reference-box>
    (Figure 8).
  </p>
  <p>
    Since H-NS is naturally present in <i>E. coli</i>, our synthetic system requires only the expression of Ler and a
    transgene
    under the control of <i>LEE5</i>. We analyzed the expression of <i>gfp</i> under the control of <i>LEE5</i> promoter as
    proof of concept.
    The resulting population was heterogeneous regarding GFP expression since one sub-population expressed GFP and the
    other didn’t (Figure 9).

  </p>


  <mat-card class="eatintotextmargin" style="overflow: auto;">
    <div class="centerstuff semiwideimg">
      <img src="http://2018.igem.org/wiki/images/f/f6/T--GO_Paris-Saclay--heterogeneity_example.png" class="wideimg">
    </div>
    <div class="mat-caption captionafterapicture">Figure 9: ‘HeteroGenious’ GFP expression using <i>LEE5</i> and Ler
      BioBricks
      <br><br>
      <div class="captionmaterialmethod">
        <mat-list>
          <mat-list-item>
            <mat-icon mat-list-icon>filter_1</mat-icon>
            <h4 mat-line>Fluorescent microscopy of <i>E. coli</i> K12 MG1655Z1 harboring pSB1C3-LEE5_GFP_native
              (BBa_K2688012)</h4>
            <p mat-line> Bacteria were cultured in LB at 37°C during 24h.</p>
          </mat-list-item>
          <mat-list-item>
            <mat-icon mat-list-icon>filter_2</mat-icon>
            <h4 mat-line>Cytometry analysis of <i>E. coli</i> K12 MG1655Z1 harboring pSB1C3-tet-Ler1 and an integrated
              LEE5-<i>gfp</i> expression cassette</h4>
            <p mat-line>Bacteria were cultured in LB containing 50 ng/ml anhydrotetracyclin at 37°C during 24h.</p>
          </mat-list-item>
        </mat-list>
      </div>
    </div>
  </mat-card>

  <p>
    Depending on which protein (H-NS or LER) cooperatively binds to the <i>LEE5</i> promoter, <i>gfp</i> expression is
    silenced or activated.
    Using this interesting property, several possibilities are opened for application to our project. Instead of <i>gfp</i>
    transgene,
    <i>cpg2</i> and <i>folC</i> encoding genes could be clones in operon under the control of <i>LEE5</i>. This way,
    part of the population will not
    express the MTX-degradation pathway and therefosre will ensure cell renewal in the bioreactor. This illustrates the
    concept of
    ‘division of labor’ between ‘cleaning’ and ‘stem-like’ bacteria. All together, we can combine the degradation
    pathway with our
    heterogeneous expression device to achieve efficient degradation and long-term maintenance of the ‘Cleaning
    Factory’.
  </p>


  <div class="formoredetailssee">For more details on parts characterization & experiments, see the <a
    (click)="this.router.navigate(['/biology/lee5'], { fragment: 'top'})">LEE5 part page</a> & <a
    (click)="this.router.navigate(['/biology/ler'], { fragment: 'top'})">Ler part page</a></div>

  <p class="spacemaker"></p>
</mat-card>
