<div class="bannercontainer">
  <img src="http://2018.igem.org/wiki/images/d/dd/T--GO_Paris-Saclay--parts_underbanner.png"
       class="bannerimg superposedbanner1">
  <img src="http://2018.igem.org/wiki/images/9/90/T--GO_Paris-Saclay--empty-banner.png"
       class="bannerimg superposedbanner2">
  <div class="headlinetext">cpg2</div>
</div>


<mat-card class="dashboard-card largemargin">
  <mat-card>
    <mat-card-header>
      <mat-card-title>Carboxypeptidase G2</mat-card-title>
      <mat-card-subtitle>cpg2 &mdash; BBa_K2688003</mat-card-subtitle>
    </mat-card-header>
    <mat-card-content>An enzyme that cleaves folates, yielding a pterydoyl ring and glutamate(s).</mat-card-content>
    <mat-card-actions>
      <a href="http://parts.igem.org/Part:BBa_K2688003" mat-raised-button color="accent">Registry page</a>
    </mat-card-actions>
  </mat-card>
  <p class="spacemaker"></p>

  <p class="intermediateconclusion">
    <b>Executive summary</b>: This BioBrick is a full translation unit encoding CPG2, an enzyme that converts the
    anticancer drug methotrexate (MTX) into less toxic compounds. When expressed in E. coli, we have shown that it can
    rapidly remove MTX from the culture medium. This is proven conclusively by HPLC, and a bioassay that measures
    residual toxicity.

  </p>
  <p class="spacemaker"></p>

  <p class="withlettrine">
    The Pseudomonas carboxypeptidase G2 (CPG2 or ‘glucarpidase’) is an hydrolase that cleaves the (poly)glutamate tail
    off folates and analogs, leaving behind a pteroate ring system
    <reference-box shorthand="Rowsell1997"></reference-box>
    . There is broad substrate
    specificity, including endogenous folates, and notably the chemotherapeutic drug MTX. Its metabolite DAMPA
    (2,4-diamino-N10-methylpteroic acid) (Figure 1) has little antifolate activity in vitro, and no relevant clinical
    effects compared to MTX
    <reference-box shorthand="Widemann2000"></reference-box>
    .
    This has led to its use as both an antidote in case of MTX poisoning
    <reference-box shorthand="Widemann2010"></reference-box>
    , and as a
    general purpose platform for novel drug delivery methods, where CPG2 would activate polyglutamated, soluble prodrugs
    in situ
    <reference-box shorthand="Masterson2006"></reference-box>
    .
    In our project MethotrExit, CPG2 is used as a biotransformation enzyme to render wastewater MTX biologically
    inactive.
  </p>

  <mat-card class="eatintotextmargin">
    <img src="http://2018.igem.org/wiki/images/e/e7/T--GO_Paris-Saclay--cpg2_react_alobe.png" class="wideimg">
    <div class="mat-caption captionafterapicture">Figure 1: The MTX biotransformation by Pseudomonas CPG2</div>
  </mat-card>

  <h1>Expression vector design</h1>

  <p>
    We engineered a two-component pathway for efficient MTX degradation with folC and cpg2 under inducible ara (AraO2
    BBa_R0081 + AraC BBa_R0080) promoter and a phage lambda pL promoter with lacO sites (BBa_R0011) lac promoter,
    respectively. The expression of their corresponding repressors (AraC and LacI) by E. coli K12 strains allows the
    inducible expression in presence of arabinose (0.2 %) and IPTG (0,5 mM).
  </p>
  <p>
    The CPG2 protein encoding sequence from Pseudomonas strain RS-16 (GenBank: M12599.2) was modified in two ways:
  </p>

  <mat-list>
    <mat-list-item>
      <mat-icon mat-list-icon>keyboard_arrow_right</mat-icon>
      <h4 mat-line>The periplasmic localisation signal was removed</h4>
      <p mat-line>Decision was taken to remove AA 1 to 22 to achieve protein sequence identity with the commercially
        available
        CPG2.</p>
    </mat-list-item>
    <mat-list-item>
      <mat-icon mat-list-icon>keyboard_arrow_right</mat-icon>
      <h4 mat-line>The sequence was codon-optimzed</h4>
      <p mat-line> For an optimal expression in <i>E. coli</i> : correcting %GC, illegal restriction site and codon
        bias.</p>
    </mat-list-item>
  </mat-list>

  <p>
    The synthetic cpg2 gene was cloned within pSB1C3, alone or associated the folC expression cassette (Figure 2).
  </p>

  <mat-card class="sidefigure">
    <img src="http://2018.igem.org/wiki/images/3/38/T--GO_Paris-Saclay--folC_cpg2_pSB1C3.png" class="wideimg">
    <div class="mat-caption captionafterapicture">Figure 2: Genetic organization of the folC-cpg2 encoding vector
      (BBa_K2688009)
    </div>
  </mat-card>

  <h1>Characterization</h1>
  <p class="intermediateconclusion">
    <b>Summary of methods</b>: We aimed to evaluate the activity of our degradation pathway, by incubating a degrader
    strain containing cpg2 encoding plasmid in LB medium containing presence of 512 µM MTX. The culture supernatant was
    harvested and filtered before assessing the residual MTX either by a bioassay (bacteria particularly sensitive to
    MTX) or HPLC.
  </p>
  <p class="intermediateconclusion">
    <b>Results summary</b>: This characterization was successful, with both the bioassay and HPLC reporting almost
    complete MTX degradation. After only 5 h of incubation with bacteria harbouring a cpg2 encoding plasmid, the MTX was
    almost completely removed from the medium (HPLC analysis).
  </p>

  <h1>Materials and methods</h1>
  <h2>Strains and plasmids</h2>

  <p>Two categories of strains were used: degrader strains (‘chassis’ harbouring cpg2 encoding plasmid), and indicative
    strains with high MTX sensitivity, as biosensor for the biological effects of MTX. All strains were derived from <i>E.
      coli</i> K12 BW25113 (<i>Δ(araD-araB)567, ΔlacZ4787(::rrnB-3), λ-, rph-1, Δ(rhaD-rhaB)568, hsdR514</i>) parental
    strain and
    obtained from the Keio mutant collection
    <reference-box shorthand="Baba2006"></reference-box>
    .
  </p>
  <p>
    For characterization purpose, two degrader strains were selected: WT and ΔtolC (Keio mutant JW5503). The latter has
    an intrinsic growth defect, but its lower MIC (minimum inhibitory concentration) for MTX
    <reference-box shorthand="Kopytek2000"></reference-box>
    made it a possibly better degrader chassis. We used ΔacrA mutant (Keio JW0452) as an indicative strain
    since it is clearly MTX-sensitive.
  </p>
  <p>We analysed MXT biotransformation by WT and ΔtolC strains harbouring pSB1C3-cpg2 (BBa_K2688003) or pSB1C3-folC-cpg2
    (BBa_K2688009). Strains containing pSB1C3-tet (BBa_R0040) or pSB1C3-folC (BBa_K2688008) were used as controls.
  </p>
  <h2>MTX biotransformation</h2>
  <p>
    The MTX degradation experiments were performed in LB medium containing 0.5 mM IPTG and 0.2% m/v arabinose
    supplemented
    with: i) 2% v/v DMSO (which is the solvent we used to prepare MTX stock solution) for the control condition,or ii)
    with 512 µM MTX in 2% DMSO (kindly provided by Raphaël Labruère).
    The degradation experiments were conducted in 15 mL Falcon tubes, with 4 mL of either control or MTX medium,
    inoculated with 1 mL of the strains at an initial OD of 0.2. The tubes were incubated for 5 to 20 h at 37°C in
    ambient
    air. Afterward, the cultures were centrifuged 15 min at 5400 g. The supernatants were filtered (0.22 µM) to
    eliminate
    cells and obtain an evaluable supernatant filtrate.
  </p>
  <p>
    To assess the quantity of MTX remaining, two assays were conducted: HPLC and MTX bioassay.
  </p>

  <h2>MTX-detection bioassays</h2>

  <p>
    The MTX bioassay was a viability experiment using the ΔacrA indicative strain cultured in 96-wells plate with 50 µL
    of
    filtrate and 50 µL ΔacrA (OD 0.04 obtained from an overnight culture in LB).
    This viability experiment was conducted in a 96-wells plate over 700 min in a Clariostar® plate reader (BMG LABtech
    GmbH), with orbital shaking, temperature control at 37°C and ambient air atmosphere, measuring OD600 every 20 min.
    The
    transparent plates were sealed with an adhesive transparent top. Duplicate or triplicate were performed for each
    condition.
  </p>
  <h2>HPLC analysis</h2>

  <p>
    The HPLC analysis was performed using a reverse phase C18 column. Detection was made using UV spectrophotometry at
    303
    nm (peak absorption 302-303 nm
    <reference-box shorthand="Merck2007"></reference-box>
    ) with peak-triggered UV-vis spectrum (diode array). The eluting solvent
    was a gradient of acetonitrile and water (with 0.1% v/v HCOOH). This mode of operation is very similar to the assay
    and impurities protocols of reference pharmacopeia for MTX
    <reference-box shorthand="PhEur2017"></reference-box>
    <reference-box shorthand="USP2007"></reference-box>
    .
  </p>

  <h1>Results</h1>
  <p>
    We focused on two aspects to evaluate the success of our project: firstly, whether the degradation process removed
    from the medium the toxicity of MTX; and secondly whether MTX was actually biotransformed as predicted. While
    related, these two goals are not identical. Indeed MTX toxicity could be removed from the medium either by
    sequestration within chassis cells or by biotransformation. The HPLC analysis directly allows the direct detection
    of MTX biotransformation, allowing a distinction between these two scenarii.
  </p>

  <h1>MTX Bioassay</h1>
  <p>
    We compared the growth of the ΔacrA indicative strain in presence of supernatants from cultures of various chassis
    incubated with MTX.
  </p>

  <mat-card class="eatintotextmargin">
    <img src="http://2018.igem.org/wiki/images/8/82/T--GO_Paris-Saclay--bioassay_res.png" class="wideimg">
    <div class="mat-caption captionafterapicture">Figure 3 : Final opacimetry (OD<sub>600nm</sub>) reached by culture of
      the &Delta;<i>acrA1</i>
      indicative strain
      <br>
      <div class="captionmaterialmethod">
        The strain were incubated with various supernatants from cultures initially containing or not MTX and strains
        harbouring plasmid encoding <i>cpg2</i>. The chassis strains (WT or &Delta;tolC) are indicated in abscissa. The
        MTX was
        initially at 514 µM and incubated during 20 h with the chassis strains. The
        supernatants were 2-fold diluted during their inoculation in the 96-plate containing the indicative strain so
        that the maximal MTX concentration is 256 µM during the bioassay. The WT strain containing pSB1C3-<i>tet</i> was
        used
        as a negative control.
      </div>
    </div>
  </mat-card>
  <p>
    When pre-treatment of the medium is done using a control strain which does not contain cpg2 transgene (“WT
    pSB1C3-tet”), the growth of the indicative strain is greatly diminished in presence of MTX. This indicates that the
    medium contained a high concentration of residual MTX: it is expected for this control condition.
  </p>
  <p>
    However, when considering the other pre-treatment conditions, we observed that the indicative strain culture reached
    the same growth yield regardless of the initial presence of MTX in the chassis culture medium. This means the
    strains harbouring cpg2 encoding plasmid have successfully removed MTX from the medium.
  </p>

  <h2>Discussion</h2>
  <p class="intermediateconclusion">
    These results demonstrate the action of the degradation pathway on the MTX toxicity.
  </p>
  <p>
    In terms of effectiveness of the various degrader strains, it seems that the co-expression of cpg2 and folC is
    associated to an better MTX removing from the medium only in the ΔtolC strain. Possibly, disabling the efflux pump
    mechanism allows a better intracellular accumulation of MTX following glutamylation by FolC.
    Nonetheless all degrader strains produce similar results in our bioassay. It may be that some are faster than other,
    but we could not assessed this property since our earliest sampling (at 5 h) has been shown by HPLC to be “too late”
    (almost all the MTX is already gone from the medium at this time point).
  </p>

  <h2>Side notes</h2>
  <p>
    In the course of running our bioassay, we noticed that the filtrates from 5 h of incubation with the chassis strains
    were more toxic than either the initial MTX containing solution or the filtrates obtained after 20 h of incubation.
    This observation is puzzling since the HPLC assay reports the MTX to have been quasi-completely eliminated at the 5
    h. Indeed the ΔacrA indicative strain is DAMPA-sensitive (data not shown) whereas its growth is not impacted by
    glutamate, the other degradation product (Figure 4). We hypothesize this effect may be related to availability of
    DAMPA in solution: it may crystallize out of solution slowly, unavailable for bacteria but still detectable by HPLC.
  </p>
  <mat-card class="eatintotextmargin">
    <img src="http://2018.igem.org/wiki/images/f/f6/T--GO_Paris-Saclay--glutamate_tox.png" class="wideimg">
    <div class="mat-caption captionafterapicture">Figure 4 : Growth curve (OD<sub>600nm</sub>) of &Delta;<i>acrA1</i>
      culture with varying concentration of glutamate
    </div>
  </mat-card>


  <h1>HPLC analysis</h1>

  <p>
    While our bioassay results prove that there is no bacteriostatic activity left in the filtrate, this does not
    conclusively prove that the MTX has been bio-transformed. We endeavoured to show this by HPLC analysis, directly
    measuring MTX and metabolite concentration.
  </p>
  <h2>Controls</h2>
  At the selected detection wavelength, there was no interference from either LB medium or DMSO (Figure 5).

</mat-card>



