<div class="bannercontainer">
  <img src="http://2018.igem.org/wiki/images/1/1b/T--GO_Paris-Saclay--tableau_banner.png" class="bannerimg superposedbanner1">
  <img src="http://2018.igem.org/wiki/images/9/90/T--GO_Paris-Saclay--empty-banner.png" class="bannerimg superposedbanner2">
  <div class="headlinetext">Project Design</div>
</div>

<mat-card class="dashboard-card largemargin">

  <h1>Project genesis</h1>

  <p class="withlettrine">
  Our team GO Paris Saclay started its journey focusing on two main projects : one named ‘Cytodestiny’ aimed to develop
    and optimize, in a yeast model, functionalized nanobodies targeting α-synuclein aggregates responsible of Parkinson's disease;
    another one, the ‘Heterogenious Cleaning Factory’ designed to develop a bacterial platform capable of cleaning up hospital wastewater
    ,contaminated with therapeutic anticancer agents. After a constructive peer-review session with our advisors and PIs,
    the ‘Heterogenious Cleaning Factory’ was finally chosen according to the following motivations :
  </p>
  <p>
  Anti-cancer drugs are chemotherapeutic agents, designed to kill or reduce proliferating cells. Most of these drugs
    are not completely metabolized by the patients, therefore are released in the environment in urine.
  </p>
  <p>
  Unfortunately, environmental risk assessment is available for only a select few anti-cancer drugs, derived mainly from predicted
    data and excluding information on their metabolites and transformation products (TPs)<reference-box shorthand="Toolaram2014"></reference-box>. It is known that
    the most commonly used drugs such as methotrexate (MTX) or 5-fluorouracil exhibit genotoxic effects due to their mechanism of action.
  </p>
  <p>The few available studies investigating these drugs show their presence in post-treatment wastewater meaning that the water
    treatment processes are not able to degrade them entirely. Here we describe how our project was designed with the purpose
    of degrading/transforming anti-cancer drugs.
  </p>

  <p>
    As proof of concept, we decided to focus on methotrexate (MTX), one of the most used anti-cancer drugs used, found in high
    concentration in untreated hospital wastewater. As a folate analog, MTX acts as a chemotherapeutic drug by inhibiting
    dihydrofolate reductase (DHFR), an enzyme that reduces folic acid to tetrahydrofolic acid. Therefore, MTX interferes with
    DNA metabolism and cell replication. It is used to treat various diseases (e.g. cancer, rheumatoid arthritis, psoriasis ...).
    Since MTX is highly toxic, its presence in the hospital wastewater is worrying.
  </p>

  <h1>Design of the methotrexate transformation pathway</h1>
  <div class="formoredetailssee">For a more detailled view, see <a (click)="this.router.navigate(['/biology','mtx'], { fragment: 'top'})">Methotrexate Degradation</a></div>

  <mat-card class="eatintotextmargin">
    <img src="http://2018.igem.org/wiki/images/f/f3/T--GO_Paris-Saclay--degrad_complete_chemical.png" class="wideimg">
    <div class="mat-caption captionafterapicture">Figure 1 : Overview of the chemical transformation produced by our pathway</div>
  </mat-card>
  <p>
    For MTX biotransformation inside the cell, we focused our attention on two enzymes that have been described as involved
    in MTX biotransformation (Figure 1). The most important one is the Pseudomonas carboxypeptidase G2 (CPG2, commercialy marketed
    as <a href="https://en.wikipedia.org/wiki/Glucarpidase">Voraxase</a><sup>tm</sup>)
    which rapidly converts MTX into less toxic metabolites glutamate and <abbr title="2,4-diamino-N10-methylpteroic acid">DAMPA</abbr>
    <reference-box shorthand="Widemann2000"></reference-box><reference-box shorthand="Larimer2014"></reference-box>. We also tested the
    interest of co-expressing a bifunctional
    dihydrofolate synthase and folylpoly-γ-glutamate synthetase (FolC) that could potentially enhance MTX catabolism by coupling
    MTX to polyglutamate<reference-box shorthand="Chabner1985"></reference-box>. By increasing its accumulation within the cell,
    FolC could allow higher effective MTX concentration and reaction rate. We hypothesized that in presence of FolC, the MTX is rapidly
    linked to polyglutamates, a reaction that could increase its metabolism by limiting its export and so that increasing the activity of CPG2.
  </p>

  <p>
    For the MethotrExit project, we introduced in our chassis an expression cassette encoding cpg2 gene under the control
    of lac promoter coupled or not to folC gene under the control of ara promoter.
  </p>

  <h2>Choice of chassis and indicative strains</h2>

  <p>
    Many laboratory strains of E. coli are resistant to MTX. This resistance is mainly due to the export of MTX through
    efflux pumps composed several proteins such as AcrA and TolC <reference-box shorthand="Kopytek2000"></reference-box>. One of the challenges of our
    project therefore relays on limiting MTX efflux to favor MTX biotransformation and sequestration within the ‘cleaning
    factory’. Moreover we needed an indicative strain to easily monitor MTX biotransformation.
  </p><p>
    We decided to take advantage of the availability in our lab of the Keio collection <reference-box shorthand="Baba2006"></reference-box> to test the
    growth of several E. coli K12 mutants (&Delta;acrA1, &Delta;acrB, &Delta;abgT, &Delta;tolC) in presence of MTX (Figure 2).
  </p>

  <p class="intermediateconclusion">
    <b>Main conclusions</b>: The WT strain may constitute a suitable ‘MTX-resistant chassis’, whereas ΔacrA1 and ΔtolC strains could constitute MTX-sensitive indicative
    strains as well as chassis accumulating MTX. Since TolC is a general efflux pump component involved in a large panel of compound efflux
    (even in standard LB growth conditions), we thought it could be interesting to also evaluate its general interest as a chassis sequestrating MTX
    (and possibly other toxic molecules). Finally, the ΔacrA1 mutant was used as an indicative strain for further bioassays since it presents no growth
    defect in absence of MTX.
  </p>

  <h1>Expression vector design</h1>
  <mat-card class="sidefigure">
    <img src="http://2018.igem.org/wiki/images/3/38/T--GO_Paris-Saclay--folC_cpg2_pSB1C3.png" class="wideimg">
    <div class="mat-caption captionafterapicture">Figure 3 : The folC-cpg2 encoding plasmid present within the MTX ‘Cleaning Factory’</div>
  </mat-card>

  <p>
    <b>FolC/folylpolyglutamate: adding a polyglutamate to MTX</b><br>
    FolC is one of the enzymes (ATP-dependent) which can drive the glutamylation reaction. FolC is naturally encoded by E. coli.
    However to control and overexpress it, we cloned folC under the control of the ara inducible promoter in pSB1C3.
  </p>
  <p>
    <b>Carboxypeptidase G2: breaking MTX into DAMPA and glutamate</b><br>
    Originating from Pseudomonas strain RS-16, carboxypeptidase is not naturally present in E. coli genome.
    The cpg2 gene was cloned within pSB1C3 under the control of lac
    inducible promoter. We constructed two plasmids: one containing only cpg2 expression cassette and the other harboring both cpg2
    and folC expression cassettes separated by a transcription terminator (Figure 3).

  </p>
  <p>
    The translation units in this expression vector were synthetized separately, courtesy of IDT, then assembled using standard biobrick assembly.
  </p>

  <h1>MTX Biotransformation Assays</h1>
  <div class="formoredetailssee">For a more detailled view, see <a (click)="this.router.navigate(['/biology','mtx'], { fragment: 'assays'})">Methotrexate Degradation : Assays</a></div>

  <p>
    To determine whether our ‘Cleaning Factories’ (WT and &Delta;tolC strains harboring cpg2 with or without folC expression cassette)
    were able to biotransform MTX, we performed the following experiments:
  </p>
  <mat-list>
    <mat-list-item>
      <mat-icon mat-list-icon>keyboard_arrow_right</mat-icon>
      <h4 mat-line>Residual toxicity bioassays</h4>
      <p mat-line> Analyzing the growth of the &Delta;acrA1 indicative strain in presence of a set of supernatants containing MTX and incubated or not with the ‘Cleaning Factories’ </p>
    </mat-list-item>
    <mat-list-item>
      <mat-icon mat-list-icon>keyboard_arrow_right</mat-icon>
      <h4 mat-line>HPLC analysis</h4>
      <p mat-line> Searching for MTX and metabolites using an analytical-grade HPLC line </p>
    </mat-list-item>
  </mat-list>

  <p>
    The experimental scheme used for this study is summarized in the Figure 4.
  </p>
  <mat-card class="eatintotextmargin" style="overflow: auto;">
    <img src="http://2018.igem.org/wiki/images/a/a4/T--GO_Paris-Saclay--recap_exp_mtx.png" style="width:40%;float:left;margin-right: 1.2em;">
    <div class="mat-caption captionafterapicture">Figure 4 : MTX-biotransformation assays
      <br><br>
      <div class="captionmaterialmethod">
        The WT and &Delta;tolC chassis strains harbouring pSB1C3-cpg2, pSB1C3-folC-cpg2 or control plasmids (pSB1C3-tet or pSB1C3-folC)
        and were precultured overnight under agitation at 37°C in 4 ml of LB supplemented with 30 µg/ml chloramphenicol.
        Thereafter, the chassis strains were inoculated at an opacimetry of 0.2 (DO600nm) in LB medium containing and 0.5 mM
        IPTG and 0.2 % arabinose and either 2% DMSO (‘control condition’) or 514 µM MTX (‘MTX condition’). Five or 24 h after
        shaking at 37°C, the samples were harvested, centrifuged and the supernatants filtered before being tested on the  &Delta;acrA1
        indicative strain or submitted to HPLC analysis. </div>
    </div>
  </mat-card>



  <p class="intermediateconclusion">
    <b>Main conclusions for the bioassay</b>: The MTX supernatants incubated during 20 h with the chassis strains (WT or ∆tolC) harboring a plasmid encoding cpg2 under
    the control of lac promoter (pSB1C3-cpg2 or pSB1C3-folC-cpg2) are not toxic anymore against the ∆acrA1 indicative strain
    (Figure 5). The best growth yields of the indicative strains are observed when the « Cleaning Factory » is the ∆tolC mutant
    harboring pSB1C3-folC-cpg2.
  </p>
  <mat-card class="eatintotextmargin">
    <img src="http://2018.igem.org/wiki/images/0/0f/T--GO_Paris-Saclay--typ_hplc_res.png" class="wideimg">
    <div class="mat-caption captionafterapicture">Figure 5 : HPLC analysis of MTX medium incubated with a control strain or a ‘MTX-Cleaning Factory’
    <br>
    <div class="captionmaterialmethod">
      The results correspond to the HPLC analysis of supernatants obtained after 5 h of incubation with WT strains harboring pSB1C3-tet
      (CLTR) or pSB1C3-folC-cpg2 (‘MTX Cleaning factory’) bacteria as described in Figure 4. The HPLC assay was run with a reverse phase,
      C18 column. Detection was made using UV spectrophotometry at 303 nm (peak absorption 302-303 nm<reference-box shorthand="Merck2007"></reference-box>) with peak-triggered
      UV-vis spectrum (diode array). The eluting solvent was a gradient of acetonitrile and water (with 0.1% v/v HCOOH). This mode of operation
      is very similar to the assay and impurities protocols of reference pharmacopeia for MTX.<reference-box shorthand="PhEur2017"></reference-box><reference-box shorthand="USP2007"></reference-box></div>
    </div>
  </mat-card>
  <p class="intermediateconclusion">
    <b>Main conclusions for the HPLC analysis</b>: MTX is removed from the culture medium in presence of bacteria harboring cpg2
    encoding plasmids as early as 5 h post-incubation. Of note, the supernatants incubated only 5h with the chassis were
    tested but still present some toxicity against the ∆acrA1 indicative strain (data not shown). We hypothesize that waiting
    longer (20 h) allow the bacteria to remove a toxic compound from the medium.
  </p>


</mat-card>









<!--
<div class="grid-container">
  <h1 class="mat-h1">Project Design</h1>

  <mat-card class="dashboard-card" >
    Test<reference-box shorthand="Test2018"></reference-box>. A new test <reference-box shorthand="Example2018"></reference-box>

    <hr>


    <h1>Methotrexate biotransformation pathway</h1>


    <h2>Chassis selection</h2>

    <h1>Heterogeneous expression system</h1>


    <svg class="lee5explainer" viewBox="0 0 300 300">
      <g
        id="layer1-7"
        inkscape:label="Layer 1"
        transform="translate(0.21045238,11.007044)">
        <path
          inkscape:connector-curvature="0"
          id="DNA_tractline"
          d="M 24.273365,68.217281 H 199.06763"
          style="fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:1.70000005;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
           />
        <rect
          y="59.248367"
          x="82.020828"
          height="15.501633"
          width="105.83334"
          id="GFP_gene"
          style="opacity:1;fill:#50b47b;fill-opacity:1;stroke:none;stroke-width:2.04960823;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1"
           />
        <path
          inkscape:connector-curvature="0"
          id="LEE5_Prom_Arrow"
          d="M 56.126599,68.217279 V 54.051998 h 9.621705"
          style="fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:1.70000005;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1;marker-end:url(#Arrow1Send)"
           />
        <text
         xml:space="preserve"
         style="font-style:normal;font-variant:normal;font-weight:300;font-stretch:normal;font-size:10.58333302px;line-height:125%;font-family:'Source Sans Pro';-inkscape-font-specification:'Source Sans Pro, Light';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;letter-spacing:0px;word-spacing:0px;writing-mode:lr-tb;text-anchor:start;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
         x="126.95313"
         y="70.528824"
         id="GFP_Label"
         ><tspan
           x="126.95313"
           y="70.528824"
           id="tspan2612"
           style="stroke-width:0.26458332px"><tspan
             x="126.95313"
             y="70.528824"
             id="tspan2610"
             style="stroke-width:0.26458332px">GFP</tspan></tspan></text>
      <text
         xml:space="preserve"
         style="font-style:normal;font-weight:normal;font-size:10.58333302px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
         x="46.903778"
         y="77.547478"
         id="LEE5_Label"><tspan
           x="46.903778"
           y="77.547478"
           id="tspan2618"
           style="stroke-width:0.26458332px"><tspan
             x="46.903778"
             y="77.547478"
             style="font-style:italic;font-variant:normal;font-weight:300;font-stretch:normal;font-size:10.58333302px;font-family:'Source Sans Pro';-inkscape-font-specification:'Source Sans Pro, Light Italic';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;writing-mode:lr-tb;text-anchor:start;stroke-width:0.26458332px"
             id="tspan2616">LEE5</tspan></tspan></text>
        <path
          style="fill:#ff3800;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
          d="M 51.645202,28.269915 H 40.589667 l -5.918626,-5.459935 5.806954,-5.047855 h 11.055534 l -5.471928,5.047855 5.471928,5.047857 z"
          id="HNS" />
        <path
          style="fill:#ff3800;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
          d="M 51.645202,28.269915 H 40.589667 l -5.918626,-5.459935 5.806954,-5.047855 h 11.055534 l -5.471928,5.047855 5.471928,5.047857 z"
          id="HNS2" />
        <path
          style="fill:#ff3800;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
          d="M 51.645202,28.269915 H 40.589667 l -5.918626,-5.459935 5.806954,-5.047855 h 11.055534 l -5.471928,5.047855 5.471928,5.047857 z"
          id="HNS3" />
        <path
          style="fill:#ff3800;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
          d="M 51.645202,28.269915 H 40.589667 l -5.918626,-5.459935 5.806954,-5.047855 h 11.055534 l -5.471928,5.047855 5.471928,5.047857 z"
          id="HNS4" />
        <path
          style="fill:#ee9942;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
          d="m 85.950738,27.608459 h -11.05554 l -5.918624,-5.459936 5.806954,-5.047855 h 11.05553 c 0.08057,3.365533 0.03878,6.730179 0,10.095713 z"
          id="Ler" />


        <path
          style="fill:#216398;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
          d="m 159.90191,38.635006 h -17.37252 c -8.7785,-1.214539 -7.30392,-5.148532 -9.35443,-8.285353 -8.62849,2.681416 -7.43344,0.162134 -10.69078,0 -5.48854,-9.664339 -5.93783,-14.587134 0.66817,-10.022609 14.65528,0 23.60451,-8.559072 43.96584,0 0.38289,7.845351 -1.89798,14.025855 -7.21628,18.307962 z"
          id="RNA_Pol" />
        <path
          style="fill:none;fill-opacity:1;fill-rule:evenodd;stroke:#000000;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:12.00000033, 2.00000006;stroke-dashoffset:0;stroke-opacity:1"
          d="M 133.69158,10.972607 120.86266,2.9545174 h -16.43709 v -4.9444776 h 16.43709 l 12.82892,7.4067856 z"
          id="RNA" />

        <text
         x="32.673473"
         y="14.795038"
         id="HNSLabel"
         >H-NS</text>

      <text
         x="71.794014"
         y="13.472123"
         id="LerLabel">Ler</text>

        <text
          x="71.794014"
          y="13.472123"
          id="RNAPolLabel">RNA Polymerase</text>


      </g>
    </svg>

  </mat-card>
</div>

-->
